src/main/webapp/dev/allJsUnitTests.jsp

```diff
@@ -298,6 +298,7 @@
             </tbody>
         </table>
     </div>
+    <button id="loading-indicator"></button>
```

Comment:
It's not advisable to keep adding new html unless it is really needed.

Take a look at using [fixtures](http://qunitjs.com/cookbook/#keeping-tests-atomic) to inject the button instead.